test/parallel/test-parse-args.mjs

```diff
@@ -1062,3 +1062,131 @@ test('auto-detect --no-foo as negated when strict:false and allowNegative', () =
   process.argv = holdArgv;
   process.execArgv = holdExecArgv;
 });
+
+test('help arg value config must be a string', () => {
+  const args = ['-f', 'bar'];
+  const options = { foo: { type: 'string', short: 'f', help: 'help text' } };
+  const help = true;
+  assert.throws(() => {
+    parseArgs({ args, options, help });
+  }, /The "help" argument must be of type string/
+  );
+});
+
+test('help value for option must be a string', () => {
+  const args = [];
+  const options = { alpha: { type: 'string', help: true } };
+  assert.throws(() => {
+    parseArgs({ args, options });
+  }, /"options\.alpha\.help" property must be of type string/
+  );
+});
+
+test('when help arg with help value for lone short option is added, then add help text', () => {
```

Comment:
The name of the test seems to be describing the arguments to be parsed rather than the interesting variations of the formatted help. I might be misunderstanding the intent.

Perhaps:

```suggestion
test('when option has short and long flags, then both appear in usage', () => {
```

Reply:
Was `lone` perhaps referring to there only being one option? Single option is fine as testing one thing at a time is good for unit tests.

I think things of interest to test are:
- option with no short flag
- option with long and short flag
- string option
- boolean option
- option without help text
- long option that causes wrap

You are testing all of these (so good coverage), but the test names seem to be describing the supplied `args` rather than the supplied `options`.

Reply:
Thank you, I pushed some improvements on the text following your suggestions!